app/services/PaypalService.scala

```diff
@@ -137,7 +138,7 @@ class PaypalService(config: PaypalApiConfig, contributionData: ContributionData)
         created = created,
         email = payerInfo.getEmail,
         ophanId = ophanId,
-        abTests = chosenVariants.asJson,
+        abTests = Json.toJson(variant),
```

Comment:
No, that makes sense – I actually wrap it in a `Seq` in the template, since all the JS stuff is written to handle arrays, but that's something that should happen further up.